.github/workflows/performance.yml

```diff
@@ -148,8 +141,7 @@ jobs:
         env:
             COMMITTED_AT: ${{ steps.commit-timestamp.outputs.result }}
             CODEHEALTH_PROJECT_TOKEN: ${{ secrets.CODEHEALTH_PROJECT_TOKEN }}
-            # TODO: We'll need to update this to point to the production dashboard once it's ready for us to start pushing data into it.
-            HOST_NAME: "codehealth-test.vercel.app"
+            HOST_NAME: "codehealth.vercel.app"
         # The base hash hard coded here is the commit to the WP 6.1.1 tag.
         run: node ./tests/performance/log-results.js $CODEHEALTH_PROJECT_TOKEN trunk $GITHUB_SHA 031cedc06d612c4a74f7b6d3e77d8145f2954a25 $COMMITTED_AT $HOST_NAME
```

Comment:
We'll need to add an `if` here so this only runs on pushes to WordPress branches, not on PRs.

Reply:
Added `if` condition in https://github.com/10up/wordpress-develop/pull/41/commits/e66687a08628d2348048dba0ec6cddc19a6d2937